src/Artsy/Router/RouterLink.tsx

```diff
@@ -41,8 +42,22 @@ export const RouterLink: React.FC<LinkProps> = ({ to, children, ...props }) => {
       ...handlers,
     ])
 
+    // Wrap any provided onClick with pageview tracking
+    // if the page type stays the same. When linking across apps
+    // those components will get mounted with their own tracking.
+    // But when linking within the same app (just different params),
+    // we want to trigger a pageview.
+    const onClick = e => {
+      const toPath = to.toString()
+      const currentPageType = window.location.pathname.split("/")[1]
```

Comment:
Can you pluck `location.pathname` from the router stored in our `SystemContext` via `useSystemContext`? We should avoid the mixing window with router instance state if possible.

Reply:
Will look into that!

Reply:
Looking at the file I just noticed [this too](https://github.com/artsy/reaction/blob/833168be194c3ed855c5953a64594e69b8e479ad/src/Artsy/Router/RouterLink.tsx#L19) -- so maybe that's already in scope?

Reply:
Ah yes, totally.